### ISSUE TITLE
pdparams and pdiparams

### ISSUE THREAD

**Issue by wario-wong** [deepinsight/insightface#1658]
The models download link give us pdiparams but the code is using pdparams.
Can you export pdparams for us to do inference?

**Comment by grasswolfs** [comment]
Could we have more details about the download link which you need, please? I am not so sure which one is you mentioned. T.T

**Comment by littletomatodonkey** [comment]
Hi, the pdparams file is the pretrained model, which is generated during the training process, the pdiparams file is generate from the pdparams file, which is suggested for the inference process. Model export is supported for both the recognition and the detection model.

**Comment by wario-wong** [comment]
Hello!! Thank you for the answer. I have more understanding now. Meanwhile, can you tell me how to set the config file for training?

**Comment by wario-wong** [comment]
Seems that we have to set directories path for training data but I don't get much detail from readme file.

**Comment by littletomatodonkey** [comment]
Hi, if you want to set the training data dir or the epochs info, you can modify file: `recognition/arcface_paddle/config.py`. If you want to modify other params such as output_dir or learning rate, you can refer to https://github.com/deepinsight/insightface/blob/master/recognition/arcface_paddle/README_ch.md#4-%E6%A8%A1%E5%9E%8B%E8%AE%AD%E7%BB%83

**Comment by littletomatodonkey** [comment]
We will also unify the config file in the future, thanks for your attention.

**Comment by wario-wong** [comment]
Thanks a lot Sir.
